Tests/Test_BlinkerLight.c

```diff
@@ -0,0 +1,55 @@
+#include "BlinkerLight.h"
+
+int main(void){
+    OS_ERR err;
+    OSInit(&err);
+
+    if(err != OS_ERR_NONE){
+        printf("OS error code %d\n", err);
+    }
+    
+    OSTaskCreate(
+        (OS_TCB*)&ReadSwitches_TCB,
+        (CPU_CHAR*)"ReadSwitches",
+        (OS_TASK_PTR)Task_ReadSwitches,
+        (void*)NULL,
+        (OS_PRIO)6,
+        (CPU_STK*)ReadSwitches_Stk,
+        (CPU_STK_SIZE)WATERMARK_STACK_LIMIT/10,
+        (CPU_STK_SIZE)TASK_READ_SWITCHES_STACK_SIZE,
+        (OS_MSG_QTY)NULL,
+        (OS_TICK)NULL,
+        (void*)NULL,
+        (OS_OPT)(OS_OPT_TASK_STK_CLR),
+        (OS_ERR*)&err
+    );
```

Comment:
It doesn't seem like you're passing in the global car state struct anywhere. Is it declared at all anywhere?